src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -760,6 +760,10 @@ var adyenHelperObj = {
       return paymentMethod === constants.PAYMENTMETHODS.APPLEPAY;
     },
 
+    isCashApp(paymentMethod) {
```

Comment:
This method would not be needed anymore when creating an array for payment method that can skip summary page